src/app/collection-page/collection-page-routes.ts

```diff
@@ -85,9 +85,13 @@ export const ROUTES: Route[] = [
         component: ThemedCollectionPageComponent,
         children: [
           {
-            path: '',
+            path: 'search',
```

Comment:
This change seems "odd" because it appears to be removing the default path for Collections (e.g. `[dspace.ui.url]/collections/[uuid]`). This change may require further testing to ensure that this small change doesn't break links to the collection homepage.